randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -3238,8 +2687,8 @@ Extra - asset_id: collision_camera_039
           Any of the following:
               # Movement is for shooting the block while on the ledge, pseudo is performed in the little alcove under the block with a melee pseudo
               Movement (Beginner) or Pseudo-Wave Beam (Beginner) or Lay Bomb or Lay Power Bomb or Shoot Diffusion or Wave
-  > Tile Group (WEIGHT)
-      Morph Ball
+  > Start Point
+      Trivial
```

Comment:
This should be morph ball in the off chance you dont have morph in a random start/teleport seed